docs/source/scheduling-state.rst

```diff
@@ -375,12 +385,29 @@ Notes:
 1. **nbytes**: a task can be in this collection as long as it was already
    computed, even if not currently held in a worker's memory.
 
+Transitions and worker state
+''''''''''''''''''''''''''''
+
+The table below shows which worker state variables are updated on each
+task state transition.
+
+==================================== ==========================================================
+Transition                           Affected worker state
+==================================== ==========================================================
+released → waiting                   occupancy, idle
+waiting → processing                 occupancy, idle, used_resources
+waiting → memory                     idle, worker_bytes
+processing → memory                  occupancy, idle, used_resources, worker_bytes
+processing → erred                   occupancy, idle, used_resources
+processing → released                occupancy, idle, used_resources
+memory → released                    worker_bytes
+memory → forgotten                   worker_bytes
```

Comment:
The task in `tasks` itself is also forgotten if you want to include this

Reply:
I am actually a bit surprised that the right column of this table is not more dense.  I would have expected more worker and task state to be affected.  Ah, I see that you are only listing worker state, never mind.